source/globalCommands.py

```diff
@@ -1703,35 +1703,92 @@ def script_reportClipboardText(self,gesture):
 	script_reportClipboardText.category=SCRCAT_SYSTEM
 
 	def script_review_markStartForCopy(self, gesture):
-		self._copyStartMarker = api.getReviewPosition().copy()
+		reviewPos = api.getReviewPosition()
+		startMark = reviewPos.copy()
+		reviewPos.obj._copyStartMarker = startMark # represents the start location

```

Comment:
The variable was just a leftover remnant from an earlier approach, I'll remove it.